### DISCUSSION TITLE
Auto delete expired links

### DISCUSSION THREAD

**Discussion by gltched-usr** [shlinkio/shlink#1710] [Help wanted] [answered]
Does shlink have a setup to automaticly delete expired links. I create some temporary links over the api that act like a sso links(but its not for login) with `validUntil`). But this leaves me with many expired links every day. Can they automaticly be deleted?

**Comment by acelaya** [comment, marked as answer]
EDIT: As of Shlink 4.1.0, this will be more or less supported. Not fully automated, but Shlink will provide a CLI command to delete expired short URLs. See https://github.com/shlinkio/shlink/pull/2086 for details.

---

I somehow missed this, and was only aware of it after someone else asked the same question.

This is not supported at the moment.

**Reply by acelaya** [reply]
A variation of this might come eventually. See https://github.com/shlinkio/shlink/issues/2059
